kamailio/datadog_checks/kamailio/check.py

```diff
@@ -0,0 +1,96 @@
+import json
+import re
+import subprocess
+
+import psutil
+
+from datadog_checks.base import AgentCheck
+
+
+class KamailioCheck(AgentCheck):
+    # this will be the prefix of every metric and service check the integration sends
+    __NAMESPACE__ = 'kamailio'
+
+    def __init__(self, name, init_config, instances):
+        super(KamailioCheck, self).__init__(name, init_config, instances)
+
+    def check(self, instance):
+        """
+        Run the kamailio service checks and send metrics to datadog API
+
+        :param instance:    The current instance configuration
+        :type instance:     object|dict
+        :return:            None
+        :rtype:             None
+        """
+
+        # validate instance configs
+        jsonrpc_api_url = instance.get('jsonrpc_api_url')
+        if jsonrpc_api_url is None:
+            self.service_check(
+                "services_up", AgentCheck.WARNING, message='agent misconfiguration: jsonrpc_api_url is not valid'
+            )
+            return None
```

Comment:
See https://github.com/DataDog/integrations-extras/pull/1610/files#r1031302293

Reply:
resolved in cf85a7a2cd46cc4e6c399e639ff477102a24aead